frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -387,7 +387,7 @@ const plugin: Plugin<ConsumedExtensions> = [
       icon: <CodeIcon />,
       getLandingPageURL: () => '/topology',
       getK8sLandingPageURL: () => '/add',
-      getImportRedirectURL: (project) => `/topology/ns/${project}`,
+      getImportRedirectURL: (project) => `/topology/ns/${project}/graph`,
```

Comment:
Remove this change. We want the user to be redirected to their previous view whether it be the `list` or `graph`.